packages/medusa/src/migrations/1646832251674-add_line_item_adjustments.ts

```diff
@@ -0,0 +1,242 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
Please clean this one up so it only includes the operations that are relevant to adjustments

Reply:
Also make sure to remoev the `"public"` schema

Reply:
right! sorry about that 😅